setup.cfg

```diff
@@ -0,0 +1,27 @@
+[pycodestyle]
+max-line-length = 88
+exclude = .eggs,ENV,build,docs/conf.py,venv
+add-ignore = D105
+# D105: Missing docstring in magic method
+
+[yala]
+radon mi args = --min C
+pylint args = --disable==too-many-arguments,too-many-locals,too-few-public-methods,too-many-instance-attributes,no-else-return,dangerous-default-value,duplicate-code,raise-missing-from,too-many-arguments,too-many-public-methods
+linters=isort,pycodestyle,pylint # TODO enable black later to minimize changes
```

Comment:
Added `black` but I haven't formatted the rest of the code with it, I'll do it in an upcoming PR to minimize changes and facilitating the review.